eclair-core/src/test/scala/fr/acinq/eclair/channel/states/a/WaitForAcceptChannelStateSpec.scala

```diff
@@ -176,6 +176,32 @@ class WaitForAcceptChannelStateSpec extends TestKitBaseClass with FixtureAnyFunS
     awaitCond(alice.stateName == WAIT_FOR_FUNDING_INTERNAL)
   }
 
+  test("recv AcceptChannel (upfront shutdown script)", Tag(StateTestsTags.OptionUpfrontShutdownScript)) { f =>
```

Comment:
We should also add tests to `WaitForOpenChannelStateSpec.scala`

Reply:
done  in https://github.com/ACINQ/eclair/pull/1846/commits/f324eae12481bcda0992c902603a01ccde580a35